support/xspec-dev/testing/xspec-test-batch.xpl

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<p:declare-step xmlns:p="http://www.w3.org/ns/xproc"
+            xmlns:c="http://www.w3.org/ns/xproc-step"
+            xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+            xmlns:xs="http://www.w3.org/2001/XMLSchema"
+            xmlns:t="http://www.jenitennison.com/xslt/xspec"
+            xmlns:mx="http://csrc.nist.gov/ns/csd/metaschema-xslt"
+            name="xspec-test-batch"
+            type="mx:xspec-test-batch"
+            version="1.0">
+   
+<!-- A model XProc showing how to run a set of XSpec documents in place,
+     showing:
+   
+   'xspec-results' - XSpec results
+   'summary' - XML-based summary
+   'determination' short synopsis in plain text (for console)
+   
+   - At a convenient location, copy and rename
+        this file
+        shell script mvn-xproc-xspec-test-batch.sh
+   
+   - rewire
+       xproc: omit any unwanted ports or serialization settings
+              update import link
+              bind XSpec documents to the input port
```

Comment:
Blocking: I am apologize but there are a  lot of moving parts here and I don't quite understand if the issue is the `p:serialization` elements or how the shell script invokes them, but I consistently get an `XS0039` error. Can we review tomorrow? I am worried maybe I missed the obvious comment, but is this a TODO before final review and merge?

```sh
[com.xmlcalabash.drivers.Main.main()] ERROR com.xmlcalabash.util.DefaultXProcMessageListener - file:/home/me/code/metaschema-xslt/branches/wendell/issue93-xproc-for-xspec_A/support/xspec-dev/xspec-batch-report.xpl:10:27:err:XS0039:A p:serialization specifies a non-existant port.
[com.xmlcalabash.drivers.Main.main()] ERROR com.xmlcalabash.drivers.Main - It is a static error if the port specified on the p:serialization is not the name of an output port on the pipeline in which it appears or if more than one p:serialization element is applied to the same port.
```

Reply:
This would have to be a bug in the XProc - will look. Maybe a port was twiddled and not re-checked....

Reply:
OK, we can look together if you would like. I spent some time looking and even thought there is a specific error code (based in spec I assume with `XS0039`) I found exactly one mailing list thread with those encountering the issue and playing with different `p:serialization` declarations in both XPL files and could not get around it. I assume it is something obvious. Let me know how I can be helped?

https://www.mail-archive.com/xproc-dev@w3.org/msg00130.html